lively.morphic/rendering/renderer.js

```diff
@@ -545,13 +559,20 @@ export default class Renderer {
    * @param {Morph} morph - The morph for which to update the scroll of its node.
    */
   updateNodeScrollFromMorph (morph) {
-    if (morph.isText && morph.document) return;
-
+    if (morph.clipMode !== 'auto' && morph.clipMode !== 'scroll') return;
     const node = this.getNodeForMorph(morph);
     if (!node) {
       return;
     }
     const { x, y } = morph.scroll;
+    if (morph.isText && morph.document) { // FIXME: maybe check if we are insisting for the document?
```

Comment:
What do you mean by insisting? That we should check if we can remove the second part of the condition?

Reply:
Yes by `morph.needsDocument`, that was my thought.